src/js/views/patients/shared/components/list-search/index.js

```diff
@@ -0,0 +1,38 @@
+import Backbone from 'backbone';
+
+import Component from 'js/base/component';
+
+import { SearchView } from './list-search_views';
+
+
+const StateModel = Backbone.Model.extend({
+  defaults: {
+    query: '',
+  },
+});
+
+export default Component.extend({
+  StateModel,
+  ViewClass: SearchView,
+  viewOptions() {
+    return {
+      model: this.getState(),
```

Comment:
So the state attributes are added to the view already, so if we don't expect the view to react to state changes then the view should probably be use those https://github.com/RoundingWellOS/marionette.toolkit/blob/master/src/component.js#L214